pehlaschoollibrary/proj.android-studio/app/src/com/maq/pehlaschool/library/SelectActivity.java

```diff
@@ -233,6 +233,7 @@ public static void updateStringLocale( Context context, String locale)
                 currConfig.setLocale(new Locale("bn"));
                 break;
             default: // Do nothing as English text is set by default
+                currConfig.setLocale(new Locale("en"));
```

Comment:
Is the comment written after default: necessary now?

Reply:
Once locale is set for an Activity it will be retained and if it is not updated/changed then the Activity instances will be created in same locale which will cause issue(unless an instance of the locale dependent activity is already created) . So if the locale is English then there is need to update the locale of the Activity.

Reply:
No, removed the comment.